src/index.js

```diff
@@ -1,40 +1,45 @@
 import Biscoint from 'biscoint-api-node';
 import _ from 'lodash';
 import player from 'play-sound';
-import config from './config.js';
 
-// read the configurations
-let {
-  apiKey, apiSecret, amount, amountCurrency, initialBuy, minProfitPercent, intervalSeconds, playSound, simulation,
-  executeMissedSecondLeg,
-} = config;
+// Declare variables
+let apiKey = process.env.apiKey
+let apiSecret = process.env.apiSecret
+let amount = process.env.amount || 100
+let amountCurrency = process.env.amountCurrency || "BRL"
+let initialBuy = process.env.initialBuy || true
+let minProfitPercent = process.env.minProfitPercent || 0.02
+let intervalSeconds = process.env.intervalSeconds || 'null'
+let playSound = process.env.playSound || false
+let simulation = process.env.simulation || false
+let executeMissedSecondLeg = process.env.executeMissedSecondLeg || true
```

Comment:
It's not common to name environment variable in **camelCase**, we usually use **ALL_CAPS_SNAKE_CASE**. But don't worry, since I'll be changing the config lib, I'll fix it.

Reply:
thas it right.. I didn't attend to it :(  It's always uppercase letters. 

Ok thank you.